actions/image_partition_action.go

```diff
@@ -481,6 +482,15 @@ func (i *ImagePartitionAction) PreNoMachine(context *debos.DebosContext) error {
 	if err != nil {
 		return fmt.Errorf("Failed to setup loop device: %v", err)
 	}
+
+	if context.SectorSize != 512 {
+		command := []string{"losetup", "--sector-size", strconv.Itoa(context.SectorSize), i.loopDev.Path()}
```

Comment:
It'd be good to add a comment as to why you're not using a library function from losetup & possibly a link to an issue requesting the feature ?

Reply:
Just added a comment and openned a pull request for go-losetup